### ISSUE TITLE
Support callbacks

### ISSUE THREAD

**Issue by reichert621** [papercups-io/chat-widget-native#5]
We currently support some callbacks (like `onMessageReceived`) in the web widget -- need to add these to the RN version as well.

(May need to leverage `window.ReactNativeWebView.postMessage(data);`)

**Comment by devanada** [comment]
yes, this could be good, waiting this to be release on RN version too
